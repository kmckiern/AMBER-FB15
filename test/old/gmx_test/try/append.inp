* Append the PDBs
*

! read topology and parameter files

read rtf card name fb15.rtf
read param card name fb15.prm

! Read PSF and coordinates from file
read psf card name fb15-pro-final.psf
read coor card name fb15-pro-final.crd

! redo hydrogen coordinates for the complete structure
! coor init sele hydrogen end
hbuild

! calculate energy
energy

ioform extended

write psf card name fb15-final.psf
* Final structure
*

write coor card name fb15-final.crd
* Final structure
*


stop


