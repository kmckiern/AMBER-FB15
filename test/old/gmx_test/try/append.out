1
                 Chemistry at HARvard Macromolecular Mechanics
            (CHARMM) - Developmental Version 37b2  February 15, 2013            
       Copyright(c) 1984-2001  President and Fellows of Harvard College
                              All Rights Reserved
       Current operating system: Linux-3.8.0-35-generic(x86_64)@not0rious       
                 Created on  8/10/15 at 14:55:29 by user: kmckiern    

            Maximum number of ATOMS:     60120, and RESidues:       20040
 RDTITL> * APPEND THE PDBS
 RDTITL> *
  
 CHARMM>     
  
 CHARMM>    ! read topology and parameter files
 CHARMM>     
  
 CHARMM>    read rtf card name fb15.rtf
 VOPEN> Attempting to open::fb15.rtf::
 MAINIO> Residue topology file being read from unit  90.
 TITLE> *  AMBER CORNELL ET AL. (PARM94 & PARM99) FORCE FIELD CONVERSION
 TITLE> *  SEE: JACS (1995) 117, 5179-5197.
 TITLE> *  SEE:  J COMP. CHEM. (2000) 21, 1049-1074.
 TITLE> *  NUCLEIC ACID PART CONVERTED BY TEC3, MARCH 1997
 TITLE> *  PROTEIN PART CONVERTED BY TEC3, FEBRUARY 1999
 TITLE> *  CHANGES FOR PARM99 BY JEFF KLAUDA OCTOBER 2003
 TITLE> 23    1
 VCLOSE: Closing unit   90 with status "KEEP"
  
 CHARMM>    read param card name fb15.prm
 VOPEN> Attempting to open::fb15.prm::

          PARAMETER FILE BEING READ FROM UNIT 90
 TITLE> *  AMBER PARM99SB FORCE FIELD CONVERSION
 TITLE> *  SEE: PROTIENS: STR. FUNC. AND BIOINFO. 65, 712 (2006)
 TITLE> *
 PARMIO> NONBOND, HBOND lists and IMAGE atoms cleared.
 VCLOSE: Closing unit   90 with status "KEEP"
  
 CHARMM>     
  
 CHARMM>    ! Read PSF and coordinates from file
 CHARMM>    read psf card name fb15-pro-final.psf
 VOPEN> Attempting to open::fb15-pro-final.psf::
 MAINIO> Protein structure file being read from unit  90.
 TITLE>  * PSF
 TITLE>  *  DATE:     8/10/15     14:55:29      CREATED BY USER: kmckiern
 TITLE>  *
 PSFSUM> PSF modified: NONBOND lists and IMAGE atoms cleared.
 PSFSUM> Summary of the structure file counters :
         Number of segments      =        1   Number of residues   =       61
         Number of atoms         =      927   Number of groups     =       57
         Number of bonds         =      934   Number of angles     =     1688
         Number of dihedrals     =     2465   Number of impropers  =      175
         Number of cross-terms   =        0
         Number of HB acceptors  =      100   Number of HB donors  =      108
         Number of NB exclusions =        0   Total charge =   -2.00000
 VCLOSE: Closing unit   90 with status "KEEP"
  
 CHARMM>    read coor card name fb15-pro-final.crd
 VOPEN> Attempting to open::fb15-pro-final.crd::
          SPATIAL COORDINATES BEING READ FROM UNIT 90
 TITLE> * COORDS
 TITLE> *  DATE:     8/10/15     14:55:29      CREATED BY USER: KMCKIERN
 TITLE> *
 VCLOSE: Closing unit   90 with status "KEEP"
  
 CHARMM>     
  
 CHARMM>    ! redo hydrogen coordinates for the complete structure
 CHARMM>    ! coor init sele hydrogen end
 CHARMM>    hbuild

      ***** LEVEL  4 WARNING FROM <GTHBUI> *****
      ***** Zero selection specified
      ******************************************
      BOMLEV (  0) IS NOT REACHED. WRNLEV IS  5

  
 CHARMM>     
  
 CHARMM>    ! calculate energy
 CHARMM>    energy

 NONBOND OPTION FLAGS: 
     ELEC     VDW      GROUps   CDIElec  SWITch   VGROup   VSWItch 
     BYGRoup  NOEXtnd  NOEWald 
 CUTNB  = 14.000 CTEXNB =999.000 CTONNB = 10.000 CTOFNB = 12.000
 CGONNB =  0.000 CGOFNB = 10.000
 WMIN   =  1.400 WRNMXD =  0.500 E14FAC =  0.833 EPS    =  1.000
 NBXMOD =      5
 There are        0 atom  pairs and        0 atom  exclusions.
 There are        0 group pairs and        0 group exclusions.
 <MAKINB> with mode   5 found   2622 exclusions and   2437 interactions(1-4)
 <MAKGRP> found     56 group exclusions.
 Generating nonbond list with Exclusion mode = 5
 == PRIMARY == SPACE FOR   266374 ATOM PAIRS AND     1238 GROUP PAIRS

 General group nonbond list generation found:
      790 GROUP PAIRS WERE FOUND FOR GROUP LIST
      863 GROUP PAIRS WERE BEYOND CUTOFFS
        0 GROUP PAIRS WERE NOT SELECTED
        0 QM group pairs excluded
ENER ENR:  Eval#     ENERgy      Delta-E         GRMS
ENER INTERN:          BONDs       ANGLes       UREY-b    DIHEdrals    IMPRopers
ENER EXTERN:        VDWaals         ELEC       HBONds          ASP         USER
 ----------       ---------    ---------    ---------    ---------    ---------
ENER>        0 204202.24129      0.00000    256.74646
ENER INTERN>      103.09769   3365.89233      0.00000 200683.95858   1799.32158
ENER EXTERN>     -215.21640  -1534.81249      0.00000      0.00000      0.00000
 ----------       ---------    ---------    ---------    ---------    ---------
  
 CHARMM>     
  
 CHARMM>    ioform extended
 MISCOM> Expanded I/O format is used.
  
 CHARMM>     
  
 CHARMM>    write psf card name fb15-final.psf
 VOPEN> Attempting to open::fb15-final.psf::
 RDTITL> * FINAL STRUCTURE
 RDTITL> *
 VCLOSE: Closing unit   90 with status "KEEP"
  
 CHARMM>     
  
 CHARMM>    write coor card name fb15-final.crd
 VOPEN> Attempting to open::fb15-final.crd::
 RDTITL> * FINAL STRUCTURE
 RDTITL> *
 VCLOSE: Closing unit   90 with status "KEEP"
 VCLOSE: Closing unit   90 with status "KEEP"
  
 CHARMM>     
  
 CHARMM>     
  
 CHARMM>    stop
$$$$$$  New timer profile $$$$$
   List time                       0.00 Other:            0.00
      Nonbond force                   0.00 Other:            0.00
         Bond energy                     0.00 Other:            0.00
         Angle energy                    0.00 Other:            0.00
         Dihedral energy                 0.00 Other:            0.00
         Restraints energy               0.00 Other:            0.00
      INTRNL energy                   0.00 Other:            0.00
   Energy time                     0.00 Other:            0.00
 Total time                      0.08 Other:            0.08

                    NORMAL TERMINATION BY NORMAL STOP
                    MOST SEVERE WARNING WAS AT LEVEL  4

                    $$$$$ JOB ACCOUNTING INFORMATION $$$$$
                     ELAPSED TIME:     0.08  SECONDS 
                         CPU TIME:     0.08  SECONDS 
